test/assembly/DockerTest.java

```diff
@@ -40,7 +40,7 @@ public void bootup() throws InterruptedException, TimeoutException, IOException
                 "docker", "run", "--name", "typedb",
                 "--rm", "-t", "-p", String.format("%d:%d", typeDBPort, typeDBPort),
                 "bazel:assemble-docker",
-                "/opt/typedb-all-linux-x86_64/typedb", "server", "--diagnostics.reporting.errors=false", "--diagnostics.reporting.statistics=false"
+                "/opt/typedb-all-linux-x86_64/typedb", "server", "--development-mode.enable=true"
```

Comment:
We run `DockerTest` and `AptTest` only with bazel, so it will be in the `development-mode` until we give it the `--//server:config=release` flag. I thought about leaving this opportunity, but I am not comfortable that somebody would like to adapt these tests to non-Bazel usage and, while the config is not explicitly set here, forget about this mode. 

In the end, I am indifferent to this flag being set inside the code for these two tests. I'm ready to remove it if you want to.

Reply:
I think this is fine